lib/diagnostic.rb

```diff
@@ -38,7 +47,9 @@ def example
 # GET /examples/:id routes to the examples#show action.
 
 RSpec.describe 'routes for examples' do
-  # your test(s) here
+  it 'routes GET /examples/:id to the examples#show action' do
+    expect(get('/examples')).to route_to('examples#show')
+  end
```

Comment:
When we have a route to test with route params, we need to route_to our controller actions with something like the following:

```
  it 'routes GET /examples/:id to the examples#show action' do
    expect(get('/examples/1')).to route_to(
      controller: 'examples',
      action: 'show',
      id: '1'
    )
  end
```